dev/provider_packages/PROVIDER_INDEX_TEMPLATE.rst.jinja2

```diff
@@ -55,6 +55,22 @@ for the minimum Airflow version supported) via
 Requirements
 ------------
 
+The minimum Apache Airflow version supported by this provider package is ``{{ MIN_AIRFLOW_VERSION }}``.
+
+{%- if PREINSTALLED_PROVIDER %}
+
+This provider package is preinstalled by default when Apache Airflow is installed. You do not need to
```

Comment:
cc: @eladkal This note will be automatically added to requirements of all preinstalled packages.